backend/db/utils.py

```diff
@@ -565,6 +564,45 @@ def get_cset_members_items_rows(
     return get_objs_by_composite_key(con, 'cset_members_items', ['codeset_id', 'concept_id'], objs)
 
 
+def fix_jagged_rows(rows: List[Dict]) -> List[Dict]:
+    """# Fix possible jaggedneess / missing fields in rows"""
+    fields = []
+    for row in rows:
+        fields.extend(row.keys())
+        for k in row.keys():
+            fields.append(k)
+    fields = set(fields)
+    return [{field: row.get(field, None) for field in fields} for row in rows]
+
+
+def update_from_dicts(con: Connection, table: str, rows: List[Dict]):
```

Comment:
## Update function (table objects)
Pass table name and a list of table objects.

Will update any rows matching the primary keys in the objects.